plugins/preset-chart-xy/tsconfig.json

```diff
@@ -29,9 +29,6 @@
     },
     {
       "path": "../../packages/superset-ui-core"
-    },
-    {
-      "path": "../../packages/superset-ui-dimension"
     }
   ]
 }
```

Comment:
This file is autogenerated by nimbus so I wouldn't worry much about the missing EOF.